ports/aubio/CMakeLists.txt

```
@@ -10,39 +10,51 @@ add_definitions(
     -DHAVE_STDARG_H=1
     -DHAVE_C99_VARARGS_MACROS=1
 
-    -DHAVE_SNDFILE=1
-    -DHAVE_WAVWRITE=1
-    -DHAVE_WAVREAD=1
-    -DHAVE_LIBAV=1
-    -DHAVE_SWRESAMPLE=1
-
     -D_CRT_SECURE_NO_WARNINGS=1
 )
 
+if(tools IN_LIST FEATURES)
```

```suggestion
if(BUILD_TOOLS)
```
I think there should be `BUILD_TOOLS`.